crates/pallets/starknet/src/lib.rs

```diff
@@ -1179,14 +1191,15 @@ impl<T: Config> Pallet<T> {
                     results.push(SimulatedTransaction {
                         transaction_trace,
                         fee_estimation: FeeEstimate { gas_consumed, gas_price, overall_fee },
-                    })
+                    });
                 }
                 Err(e) => {
-                    log::error!("Failed to simulate transactions: {:?}, error: {:?}", tx, e);
+                    log::error!("Failed to simulate transaction: {:?}, error: {:?}", tx, e);
```

Comment:
Why do we log here if we also return the error? Is it just for debugging?

I don't think we want our node to log an error everytime an use simulate an invalid tx

Reply:
Removed log here